automatic/nexus-repository-migrator/tools/ConvertFrom-NexusOrientDb.ps1

```diff
@@ -0,0 +1,128 @@
+﻿<#
+  .Synopsis
+    Migrates Sonatype Nexus Repository 3.70.1-02 to the H2 database type
+
+  .Example
+    .\ConvertFrom-NexusOrientDb.ps1
+
+  .Example
+    .\ConvertFrom-NexusOrientDb.ps1 -JavaPath .\path\to\java.exe
+#>
+[CmdletBinding(SupportsShouldProcess)]
+param(
+  [string]$JavaPath,
+
+  [switch]$Force
+)
+
+Import-Module $PSScriptRoot\nexus-helpers.ps1 -Verbose:$false
```

Comment:
Where is this file?

Reply:
The nuspec pulls the helpers file from nexus repository. I had written it to use it as psm1, but it turned out that NuGet treats file extension changes as a folder instead.